packages/backend/server/src/data/migrations/1745211351719-create-indexer-tables.ts

```diff
@@ -0,0 +1,14 @@
+import { ModuleRef } from '@nestjs/core';
+import { PrismaClient } from '@prisma/client';
+
+import { IndexerService } from '../../plugins/indexer';
+
+export class CreateIndexerTables1745211351719 {
```

Comment:
This needs to be changed to execute every time, otherwise switching from ms to es after the self-host deployment will not automatically create the index table.